src/workloads/readme.md

```diff
@@ -0,0 +1,2 @@
+# Building
+The workloads project can only be built using .NET Framework msbuild. To build locally, run ```build -project src\workloads\workloads.csproj -msbuildEngine vs```
```

Comment:
Did you copy this from my dotnet/installer PR? That was true before we rebuilt SWIX to support dotnet build and not depend on msbuild.

We should remove this readme